packages/meditrak-server/src/routes/exportSurveyResponses/exportSurveyResponses.js

```diff
@@ -140,8 +140,8 @@ export async function exportSurveyResponses(req, res) {
           answers.forEach(({ 'question.id': questionId, text }) => {
             answersByQuestionId[questionId] = text;
           });
-          surveyResponseAnswers.push(answersByQuestionId);
         }
+        surveyResponseAnswers.push(answersByQuestionId);
```

Comment:
Fixes a problem introduced [here](https://github.com/beyondessential/tupaia/commit/429ac41c5ceb98927a016b031ea0a2cdd06366cf#diff-a647e9436011e3e8b62b5c397d33aba05d78c84f55affb9f513daa32a1f26a92R140-R145) which would only add the answers object to columns that had at least one answer defined